src/main/java/seedu/tr4cker/logic/commands/CommandResult.java

```diff
@@ -56,6 +59,19 @@ public CommandResult(String feedbackToUser, LocalDate localDate, YearMonth yearM
         this.yearMonth = yearMonth;
     }
 
+    /**
+     * Constructs a {@code CommandResult} with the specified {@code feedbackToUser}.
+     * For usage of CountdownCommand.
+     */
+    public CommandResult(String feedbackToUser, boolean isShowCountdown) {
```

Comment:
Would it be better to include `this.showPlanner = false;` too?
Same as the overloaded constructor at the top too. e.g. `this.showCountdown = false;`